packages/tradershub/src/constants/constants.tsx

```diff
@@ -23,40 +23,40 @@ export type IconToCurrencyMapperType = {
 export const optionsAndMultipliersContent = (isEU: boolean) => [
     {
         description: isEU ? 'Multipliers trading platform.' : 'Options and multipliers trading platform.',
-        icon: <PlatformIcon icon='DTrader' />,
+        icon: <IconComponent icon='DTrader' />,
         redirect: '/',
-        smallIcon: <PlatformIcon height='32px' icon='DTrader' width='32px' />,
+        smallIcon: <IconComponent height='32px' icon='DTrader' width='32px' />,
```

Comment:
let's change the height and width to numbers

Reply:
can use tailwind class or just numbers here?